### PR TITLE
empty areas dropzones should take up the full placeholder of the area

### PR DESCRIPTION
**Pull request by stuartromanek**

### CODE REVIEW

**Comment by stuartromanek** [comment]
This
http://g.recordit.co/HcR9AzL4Wi.gif

versus this
http://g.recordit.co/nhwlXdkIIB.gif

File: lib/modules/apostrophe-areas/public/css/user.less [hunk 1]
```diff
@@ -142,16 +142,26 @@
   opacity: 1;
   padding: @apos-padding-1;
   margin: @apos-margin-1/2 0;
+  // height: 100px;
```
Comment (bobclewell) [inline lib/modules/apostrophe-areas/public/css/user.less]: comment. remove?

File: lib/modules/apostrophe-areas/public/css/user.less [hunk 2]
```diff
@@ -142,16 +142,26 @@
...
 }
 
 .apos-dragging .apos-empty .apos-area-widgets
 {
+
```
Comment (bobclewell) [inline lib/modules/apostrophe-areas/public/css/user.less]: blank line.

File: lib/modules/apostrophe-areas/public/css/user.less (hunk 1 above)
Comment (stuartromanek) [reply lib/modules/apostrophe-areas/public/css/user.less]: removed

File: lib/modules/apostrophe-areas/public/css/user.less (hunk 2 above)
Comment (stuartromanek) [reply lib/modules/apostrophe-areas/public/css/user.less]: cleaned up

**Pull request merged**